### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nityam01**
Add Project Setup steps to setup noodle locally

### CODE REVIEW

**Comment by Devesh-N** [comment]
@nityam01 I have made Docker file so can I merge those changes?

File: README.md [hunk 1]
```diff
@@ -40,6 +40,26 @@ With Noodle being open source, we hope to build a community of students and deve
 - 📈 Insights
 - 🧠 Mind maps
 
+## Project Setup
+#### Pre-requisites
```
Comment (Eldemarkki) [inline README.md:44]: There shouldn't be a jump from `<h2>` directly to `<h4>`, so I propose that we change the `####` headings to just `###` to keep it semantically correct.

See more: https://www.w3.org/TR/2012/NOTE-WCAG20-TECHS-20120103/G141#G141-description

**Review by Eldemarkki — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nityam01) [reply README.md]: Thanks, I have fixed it

**Comment by ixahmedxi** [comment]
The local running of noodle is much more complex than that, purely from the fact that a database has to be in connection, oauth tokens for authentication and so on. These guidelines will be added to the CONTRIBUTING.md file once the project structure has been made and after that there will be a dedicated Noodle documentation website that will host everything someone needs to contribute and help with the development of Noodle.

I appreciate the effort, however this is not the kind of instructures we are looking for.

**Comment by nityam01** [comment]
Thank you for the feedback. I'm wondering if there is a possibility for me to contribute to Noodle. Is there a project board available where we can explore and identify the features we can collaborate on?

**Pull request closed without merging**